trie/hexary.py

```diff
@@ -182,33 +181,28 @@ def get_proof(self, key):
         node = self.get_node(self.root_hash)
         trie_key = bytes_to_nibbles(key)
 
-        return self._get_proof(node, trie_key)
+        if node == BLANK_NODE:
+            return (node,)
```

Comment:
I think we can drop this special case handling by changing `_get_proof`. Commented below.